Admin.js

```diff
@@ -34,78 +35,99 @@ var Admin = {
+        jQuery(".modal-body", modal).css({
+            "width": "auto",
+            "height": "90%",
+            "padding": 15,
+            "overflow": "auto"
         });
 
-        jQuery(modal).trigger('sonata-admin-setup-list-modal');
+        jQuery(modal).trigger("sonata-admin-setup-list-modal");
+
     },
-    setup_select2: function(subject) {
+    "setup_select2" (subject) {
+
         if (window.SONATA_CONFIG && window.SONATA_CONFIG.USE_SELECT2) {
-            Admin.log('[core|setup_select2] configure Select2 on', subject);
 
-            jQuery('select:not([data-sonata-select2="false"])', subject).each(function() {
-                var select            = jQuery(this);
-                var allowClearEnabled = false;
-                var popover           = select.data('popover');
+            Admin.log("[core|setup_select2] configure Select2 on", subject);
+
+            jQuery("select:not([data-sonata-select2=\"false\"])", subject).each(function () {
 
-                select.removeClass('form-control');
+                const select = jQuery(this);
+                let allowClearEnabled = false;
+                const popover = select.data("popover");
+
+                select.removeClass("form-control");
+
+                if (select.find("option[value=\"\"]").length || select.attr("data-sonata-select2-allow-clear") === "true") {
 
-                if (select.find('option[value=""]').length || select.attr('data-sonata-select2-allow-clear')==='true') {
                     allowClearEnabled = true;
-                } else if (select.attr('data-sonata-select2-allow-clear')==='false') {
+
+                } else if (select.attr("data-sonata-select2-allow-clear") === "false") {
+
                     allowClearEnabled = false;
+
                 }
 
                 select.select2({
-                    width: function(){
+                    "width" () {
+
                         // Select2 v3 and v4 BC. If window.Select2 is defined, then the v3 is installed.
                         // NEXT_MAJOR: Remove Select2 v3 support.
                         return Admin.get_select2_width(window.Select2 ? this.element : select);
+
                     },
-                    dropdownAutoWidth: true,
-                    minimumResultsForSearch: 10,
-                    allowClear: allowClearEnabled
+                    "dropdownAutoWidth": true,
+                    "minimumResultsForSearch": 10,
+                    "allowClear": allowClearEnabled
                 });
 
                 if (undefined !== popover) {
-                    select
-                        .select2('container')
-                        .popover(popover.options)
-                    ;
+
+                    select.
+                        select2("container").
+                        popover(popover.options);
+
                 }
+
             });
+
         }
+
     },
-    setup_icheck: function(subject) {
+    "setup_icheck" (subject) {
```

Comment:
- eslint: Expected object keys to be in ascending order. 'setup_icheck' should be before 'setup_select2'.